testing/tests/DevExpress.viz.gauges/barGauge.tests.js

```diff
@@ -574,6 +574,29 @@ QUnit.test('Some values are not changed', function(assert) {
     }, this);
 });
 
+QUnit.test('resolveLabelOverlapping - shift', function(assert) {
```

Comment:
No tests on the follow cases:
1. Hide labels that cross the title, also if title is not set
2. Hide labels that not fit to the canvas
3. Check labels from left and right ( < 90 and > 90 degrees)

Reply:
Unit tests for these cases are written